### PR TITLE
Saleor 3357 tests for orders

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for orders

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3357 Create tests for orders](https://app.clickup.com/t/2549495/SALEOR-3357)

**Review by dominik-zeglen — APPROVED** [review]

File: cypress/apiRequests/Order.js [hunk 1]
```diff
@@ -57,13 +57,17 @@ export function completeOrder(orderId) {
...
       }
       orderErrors{
         message
       }
     }
   }`;
-  return cy.sendRequestWithQuery(mutation);
+  return cy.sendRequestWithQuery(mutation).its("body.data.draftOrderComplete");
 }
 export function getOrder(orderId) {
```
Comment (eCzerniak) [inline cypress/apiRequests/Order.js:72]: Please add empty lines.

File: cypress/integration/allEnv/orders/orders.js [hunk 2]
```diff
@@ -113,4 +122,67 @@ describe("Orders", () => {
         .should("not.exist");
     });
   });
```
Comment (eCzerniak) [inline cypress/integration/allEnv/orders/orders.js:124]: Pleas add an empty lines in this file.

File: cypress/utils/ordersUtils.js [hunk 3]
```diff
@@ -61,7 +61,30 @@ export function createReadyToFulfillOrder(
...
+    address
+  }).then(({ order }) => {
+    orderRequest.fulfillOrder({
+      orderId: order.id,
+      warehouse,
+      quantity,
+      linesId: order.lines
+    });
+  });
+}
```
Comment (eCzerniak) [inline cypress/utils/ordersUtils.js:87]: Please add empty lines in this file.

**Review by eCzerniak — APPROVED** [review]

File: cypress/apiRequests/Order.js [hunk 4]
```diff
@@ -78,3 +82,30 @@ export function getOrder(orderId) {
...
+export function fulfillOrder({ orderId, warehouse, quantity, linesId }) {
+  let lines = "";
+  linesId.forEach(lineId => {
+    lines += `{orderLineId:"${lineId.id}"
+      stocks:{
+        quantity:${quantity}
+        warehouse:"${warehouse}"
+      }
+    }`;
+  });
```
Comment (jwm0) [inline cypress/apiRequests/Order.js:95]: There's a method on Array type called `reduce` that does exactly the thing you want - takes an array and outputs (usually) non-Array variable. Here's how you use it:
```
// reduce((accumulator, currentValue) => { ... }, initialValue)

linesId.reduce((lines, lineId) => {
  line = `...`;
  return lines + line;
}, "")
```

**Review by jwm0 — COMMENTED** [review]
Looks good 👍

File: cypress/apiRequests/Order.js (hunk 4 above)
Comment (karola312) [reply cypress/apiRequests/Order.js:95]: I replaced forEach with reduce

File: cypress/utils/ordersUtils.js (hunk 3 above)
Comment (karola312) [reply cypress/utils/ordersUtils.js:87]: I added lines in all files you mentioned

File: cypress/integration/allEnv/homePage/homePageAnalitics.js [hunk 5]
```diff
@@ -130,7 +130,7 @@ describe("Homepage analytics", () => {
       ).should("be.visible");
     });
   });
```
Comment (eCzerniak) [inline cypress/integration/allEnv/homePage/homePageAnalitics.js]: Please add an empty line

**Review by jwm0 — APPROVED** [review]

**Pull request merged**